src/server/appserver.h

```diff
@@ -125,6 +125,7 @@ class AppServer : public SharedTools::QtSingleApplication {
         bool _vfsActivationDone = false;
         bool _vfsConnectionDone = false;
         bool _crashRecovered = false;
+        QString _crashMessage = tr("kDrive application will close due to a fatal error.");
```

Comment:
This does not need to be a attribute of the class since it never change. Make it a static const value defined in the cpp file (or wherever you find it relevant)

Reply:
I will edit this in next commit.

Reply:
[e25a99e](https://github.com/Infomaniak/desktop-kDrive/pull/77/commits/e25a99e535064117fe127658134fba60114dc153)